.eslintrc

```diff
@@ -51,10 +49,20 @@
             "ignoreVoid": true
           }
         ],
-        "@typescript-eslint/require-await": "off",
+        "require-await": "off",
+        "@typescript-eslint/require-await": "error",
         "@typescript-eslint/no-empty-function": "off",
         "@typescript-eslint/ban-types": "off" // TODO: turn on
-      }
+      },
+      "overrides": [
+        {
+          "files": ["*.test.*"],
```

Comment:
There were a _lot_ of test cases that violated this, and I don't necessarily think it's worth enforcing this for test cases. 😄